### PR TITLE
Added meta tags, tittle and favicon 

### PR DESCRIPTION
**Pull request by Robiin007**
Changes:
-Updated the title tag
- Added images  for favicon and meta banner
-  Added meta tags, including OG and Twitter tags
- Added favicon link tag.

Test cases Checked
- Verified the title showing up.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,9 +3,20 @@
...
+    <meta property="og:title" content="Jovian Careers">
+    <meta property="og:description" content="Learn about job openings at Jovian and submit your application now">
+    <meta property="og:image" content="/jovian_meta.png">
+    <meta name="twitter:title" content="Jovian Careers">
+    <meta name="twitter:description" content="Learn about job openings at Jovian and submit your application now">
+    <meta name="twitter:image" content="/jovian_meta.png">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Assignment-2</title>
+    <title>Assignment Practise</title>
```
Comment (robinn007) [inline src/index.html:19]: make the tittle longer please at least 24 characters.

**Review by robinn007 — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (Robiin007) [reply src/index.html:19]: okay understood

**Comment by Robiin007** [comment]
okay done

**Comment by Robiin007** [comment]
okay and thanks

**Review by Robiin007 — COMMENTED** [review]
approv

**Review by robinn007 — COMMENTED** [review]
done

**Review by robinn007 — COMMENTED** [review]
nice

**Review by Robiin007 — COMMENTED** [review]
okay

**Review by robinn007 — COMMENTED** [review]
done

**Review by robinn007 — COMMENTED** [review]
nice work

**Pull request closed without merging**